Packs/Druva/pack_metadata.json

```diff
@@ -1,11 +1,11 @@
 {
-    "name": "Druva",
-    "description": "Druva Integration",
+    "name": "Druva Ransomware Response",
+    "description": "Druva Ransomware Response Integration provides an API based orchestration framework for Druva Ransomware Recovery customers. The integration allows Druva customers to respond immediately in case of a security incident and recovery their backed up data with Confidence",
     "support": "partner",
     "currentVersion": "1.0.0",
     "author": "Druva",
     "url": "",
-    "email": "krishnakumar.vasudevan@druva.com",
+    "email": "support@druva.com",
```

Comment:
we added a new option for the partners to monitor changes in their pack.
you can add here a field called `githubUser` with a list value of all Github users you would like to be notified once one of the pack files is changed.